Src/AutoFixture/RangedNumberGenerator.cs

```diff
@@ -194,5 +194,14 @@ private static object Add(object a, object b)
                     throw new InvalidOperationException("The underlying type code of the specified types is not supported.");
             }
         }
+

```

Comment:
We can move this to a separate file but, according to the [rule of three](http://c2.com/cgi/wiki?RuleOfThree), we can wait.